pkg/render/apiserver.go

```diff
@@ -1284,6 +1300,24 @@ func (c *apiServerComponent) queryServerContainer() corev1.Container {
 		env = append(env, c.cfg.KeyValidatorConfig.RequiredEnv("")...)
 	}
 
+	// set Log_LEVEL for queryserver container
+	if c.cfg.APIServer.APIServerDeployment != nil && c.cfg.APIServer.APIServerDeployment.Spec != nil &&
+		c.cfg.APIServer.APIServerDeployment.Spec.Template != nil &&
+		c.cfg.APIServer.APIServerDeployment.Spec.Template.Spec != nil &&
+		c.cfg.APIServer.APIServerDeployment.Spec.Template.Spec.Containers != nil {
+		containers := c.cfg.APIServer.APIServerDeployment.Spec.Template.Spec.Containers
+		for _, con := range containers {
+			if strings.Contains(con.Name, "queryserver") {
+				if logLevel := con.LogLevel; logLevel != nil {
+					env = append(env, corev1.EnvVar{Name: "LOGLEVEL", Value: *logLevel})
+				}
+			}
+		}
+	} else {
+		// set default LOG_LEVEL to info when not set by the user
+		env = append(env, corev1.EnvVar{Name: "LOGLEVEL", Value: "info"})
```

Comment:
LOG_LEVEL?

Reply:
in queryserver, it's checking for `LOGLEVEL` and in apiserver it's checking for `LOG_LEVEL` and I just found out about it. Since we already cut the release in calico-private, I thought I keep it consistent with the variables already defined in these two services and maybe change the one in queryserver master. WDYT?
Is the operator version to calico-private release version 1-1? or do we need to be backward compatible to some versions in the operator?

Reply:
A given operator always installs a specific version of Calico, so don't need to worry about that. But if they use different vars then there's nothing to do here in this PR - just kind of annoying :sweat_smile: 

Wanted to make sure it wasn't a typo

Reply:
I should be updating the comment though!